src/divan.rs

```diff
@@ -187,57 +205,55 @@ impl Divan {
         tree: &[EntryTree],
         shared_context: &SharedContext,
         parent_options: Option<&BenchOptions>,
-        tree_painter: &RefCell<TreePainter>,
```

Comment:
We no longer need refcells here, we just take `&mut StatCollector` as a parameter to our helper closures.
Since this is a requirement for the restructured StatCollector anyways, there is no extra convenience in using `RefCell` to avoid double mutable borrow from capturing instead in the helper closures.